templates/example_fiber_segmentation.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>Returning the labeled images currently doesn't serve a purpose, and will take up a lot of memory. Let's just remove that functionality</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>